admin/nodes/serializers.py

```diff
@@ -19,6 +19,7 @@ def serialize_node(node):
         'parent': node.parent_id,
         'root': node.root._id,
         'is_registration': node.is_registration,
+        'is_stuck_registration': getattr(node, 'is_stuck_registration', False),
```

Comment:
Where are you using this?

Reply:
[here] (https://github.com/CenterForOpenScience/osf.io/pull/8271/files#diff-5aee9e54ada2eec32fe347e858285b03R50) easy to miss.